.github/pull_request_template.md

```diff
@@ -1,14 +1,32 @@
 <!--
 Thank you for your contribution to iTwinUI.
-Please describe your PR here and make sure to complete all of the items below before submitting.
+
+If you are only making changes to the docs site using the "Edit page on GitHub" link,
+then you can ignore most of this template.
+-->
+
+## Changes
+
+<!--
+What kind of code changes does this PR include?
+Mention anything that could be helpful for reviewers and include screenshots for visual changes.
+-->
+
+## Testing
+
+<!--
```

Comment:
Maybe we should add about verifying the existing tests pass? Because if we create a `v1` branch like before that does not run CI before the merge into the `v1` branch, failing tests may be merged.

Reply:
I would like to prevent this situation entirely. Tests should run in CI in all branches, and if they are not running, then we should address that instead of changing the PR template.